docs/conf.py

```diff
@@ -229,7 +243,8 @@
             'astropy': None,
             'matplotlib': 'http://matplotlib.org/',
             'numpy': 'http://docs.scipy.org/doc/numpy/',
-        }
+        },
+        'abort_on_example_error': False
```

Comment:
This probably only wants to be false on RTD? You want a travis build to fail if the examples are failing??